From: Randy Dunlap <randy.dunlap@oracle.com>
Subject: x86: k8.h: Add struct bootnode
References: fate#311968
Git-commit: 0420101c075530c65ba00b6fe7291b126fbfc5d2
Patch-mainline: v2.6.34-rc1


Signed-off-by: Thomas Renninger <trenn@suse.de>

k8.h uses struct bootnode but does not #include a header file
for it, so provide a simple declaration for it.

  arch/x86/include/asm/k8.h:13: warning: 'struct bootnode'
  declared inside parameter list arch/x86/include/asm/k8.h:13:
  warning: its scope is only this definition or declaration, which is probably not what you want

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Acked-by: David Rientjes <rientjes@google.com>
Cc: Stephen Rothwell <sfr@canb.auug.org.au>
LKML-Reference: <20091028160955.d27ccb16.randy.dunlap@oracle.com>
Signed-off-by: Ingo Molnar <mingo@elte.hu>

 arch/x86/include/asm/k8.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/x86/include/asm/k8.h b/arch/x86/include/asm/k8.h
index c092f72..f70e600 100644
--- a/arch/x86/include/asm/k8.h
+++ b/arch/x86/include/asm/k8.h
@@ -4,6 +4,7 @@
 #include <linux/pci.h>
 
 extern struct pci_device_id k8_nb_ids[];
+struct bootnode;
 
 extern int early_is_k8_nb(u32 value);
 extern struct pci_dev **k8_northbridges;
